### PR TITLE
[luci] Make negative BN gamma to positive

### PR DESCRIPTION
**Pull request by jinevening**
This adds a pass to make negative BN gamma to a small positive value (1e-10)

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: #4773
Draft PR: #4788

### CODE REVIEW

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

File: compiler/luci/pass/include/luci/Pass/MakeBatchNormGammaPositivePass.h [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#include <logo/Pass.h>
+
+namespace luci
+{
+
+/**
+ * @brief  Class to make negative gamma of batchnorm to a small positive value (1e-10)
+ *         This pass can change the execution result of the model.
+ *         So, use it only when the impact is known to be acceptable.
+ */
```
Comment (lemmaa) [inline compiler/luci/pass/include/luci/Pass/MakeBatchNormGammaPositivePass.h:29]: It doesn't have to be resolved in this PR, shouldn't this be provided as a help description of the options in `circle-optimizer` CLI?

Comment (jinevening) [reply compiler/luci/pass/include/luci/Pass/MakeBatchNormGammaPositivePass.h:29]: I agree. I'll add this description in the option of `circle-optimizer` as well.

**Pull request merged**